scripts/conp_to_nidm_terms/report_generator.py

```diff
@@ -0,0 +1,52 @@
+import getopt
+from sys import argv
+from datetime import date
+import json
+from functions import collect_values, find_duplicates
+
+
+def main(argv):
+    timestamp = date.today()
+    opts, args = getopt.getopt(argv, "", ["filename=", "privacy=", "types=", "licenses=",
+                                          "is_about=", "formats=", "keywords=", "help"])
+
+    options = dict(privacy=True, types=True, licenses=True, is_about=True, formats=True, keywords=True)
+    filename = f"report_{timestamp}"
+
+    for opt, arg in opts:
+        opt_properties = ["--privacy", "--types", "--licenses", "--is_about", "--formats", "--keywords"]
+        if opt in opt_properties and arg == "False":
+            options[opt.replace("--", "")] = False
+        elif opt == '--filename':
+            filename = arg
+        else:
+            help_info()
+            exit()
+
+    report, dats_files_count = collect_values(
+        privacy=options["privacy"],
+        types=options["types"],
+        licenses=options["licenses"],
+        is_about=options["is_about"],
+        formats=options["formats"],
+        keywords=options["keywords"]
+    )
+    print(f"DATS files processed: {dats_files_count}")
+    # create file with duplicate terms
+    with open("duplicates.txt", "w") as f:
+        for i, item in enumerate(find_duplicates(report), 1):
+            f.write(f"{i}. {item}\n")
```

Comment:
might be worth adding a print statement here to say that duplicates were found and that the duplicates.txt file has been generated.